SUMMARY.md

```diff
@@ -41,6 +41,7 @@
   * [XDM Experience Events](using-mobile-extensions/adobe-edge/xdm-experience-events.md)
   * [Edge Validation](using-mobile-extensions/adobe-edge/edge-validation.md)
 * [Consent](using-mobile-extensions/adobe-edge-consent/README.md)
```

Comment:
Not related to this PR, but we might need to add a note like this in the Consent README:
```
If you are using Adobe Experience Platform, you will need include a `Privacy/Personalization/Marketing Preferences (Consents)` mixin to your profile schema. See [Governance, privacy, and security in Adobe Experience Platform](https://experienceleague.adobe.com/docs/experience-platform/landing/governance-privacy-security/overview.html) for more information on the Adobe standard version 2.0. 
```